config/boards/rock-3c.csc

```diff
@@ -0,0 +1,12 @@
+# Rockchip RK3566 quad core 1/2GB RAM SoC WIFI/BT eMMC USB3 PCIe
+BOARD_NAME="Rock 3C"
+BOARDFAMILY="rk35xx"
+BOARD_MAINTAINER="AbuzzHarbor9999"
```

Comment:
Who is that ? :)

Reply:
Sorry,"AbuzzHarbor9999" is my another name on the Internet.Maybe I will delete it in this file.It can boot.But I didn't fully compile armbian for it.I replaced the .dtb file for an Armbian Image for panther x2 and it works well.

Reply:
O.K.
We are keeping developers contact base in order to communicate better elsewhere https://github.com/armbian/build/pull/6176 Register there or keep empty. Anyway we have commit history which shows who added this.